$NetBSD: patch-configure,v 1.4 2014/08/17 08:57:01 adam Exp $

L2TPV3 works on Linux; don't turn it on for other platforms.

--- configure.orig	2014-08-17 08:35:26.000000000 +0000
+++ configure
@@ -1722,6 +1722,7 @@ fi
 # L2TPV3 probe
 
 cat > $TMPC <<EOF
+#include <linux/ip.h>
 #include <sys/socket.h>
 int main(void) { return sizeof(struct mmsghdr); }
 EOF
